hawc/services/epa/dsstox.py

```diff
@@ -19,7 +19,7 @@ def create_from_dtxsid(cls, dtxsid: str) -> "DssSubstance":
         """Fetch a DssTox instance from the actor webservices using a DTXSID.
 
         Args:
-            dtxsid (str): a DTXSID identifer
+            dtxsid (str): a DTXSID identifier
```

Comment:
a little scope creep from me :( -  fixed a typo throughout where we spelled "identifer" instead of "identifier"